client/components/survey-modal/style.scss

```diff
@@ -95,7 +95,7 @@
 
 				.modal-survey-notice__popup-content-buttons-ok {
 					padding: 8px 14px;
-					background: var(--studio-blue-50);
+					background: var(--wp-admin-theme-color, #3858e9);
```

Comment:
I checked and we use `--color-accent-60` instead of this. It would be better to align it with other places.

Reply:
All of this custom styling is unnecessary. We can delete it all and fix the component to use [primary](https://wordpress.github.io/gutenberg/?path=/story/components-button--primary) & [tertiary](https://wordpress.github.io/gutenberg/?path=/story/components-button--tertiary) button variants.